### PR TITLE
doc: RSC service total distance correction and units

### PR DESCRIPTION
**Pull request by dchat-nordic**
Corrected:
number of bytes for set Total Distance service 3->4 (uint32)

Specified units (rscs.h) for:
-Instantenous speed,
-Instantenous Cadence,
-Instantenous Stride Length,
-Total Distance

Signed-off-by: Dominik Chat <dominik.chat@nordicsemi.no>

### CODE REVIEW

**Comment by dchat-nordic** [comment]
Removed tests, because only comments and docs are changed.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -78,7 +78,7 @@ For lists of protocol-specific changes, see `Protocols`_.
 Bluetooth samples
 -----------------
 
-* |no_changes_yet_note|
+* Updated :ref:`peripheral_rscs` - corrected amount of bytes for setting the Total Distance Value and specified the data units.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:81]:
```suggestion
* Updated:

  * :ref:`peripheral_rscs` - Corrected the number of bytes for setting the Total Distance Value and specified the data units.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (dchat-nordic) [reply doc/nrf/releases/release-notes-changelog.rst]: Added the change

**Review by peknis — APPROVED** [review]

**Review by rafpelc — APPROVED** [review]

**Review by grochu — APPROVED** [review]
Looks good.

**Review by KAGA164 — APPROVED** [review]

**Comment by peknis** [comment]
Rebase will do the trick and make the doc build pass.

**Pull request merged**